modules/RTC/adapter.screenshare.js

```diff
@@ -4013,21 +4013,24 @@ if ( (navigator.mozGetUserMedia ||
     }
 
     var downloadLink = AdapterJS.WebRTCPlugin.pluginInfo.downloadLink;
-    if(downloadLink) { // if download link
+
```

Comment:
Can you try this ? https://github.com/BeatC/lib-jitsi-meet/blob/35636308c361485b0d6b003fef810ae9e6f6bedc/modules/RTC/adapter.screenshare.js#L17